scripts/documentation/api-markdown-table-template.hbs

```diff
@@ -0,0 +1,36 @@
+{{#each modules as |module|}}
+### {{module.moduleName}}
+{{#if module.moduleName.length}}
+#### Methods
+
+Signature | Description
+--- | ---
+{{#each module.methods}}
+`{{methodSignature}}` | {{documentation}}
+{{/each}}
+
+{{/if}}
+{{#if module.properties.length}}
+#### Properties
+
+Name | Type | Description
+--- | --- | ---
+{{#each module.properties}}
+{{name}} | {{type}} | {{documentation}}
+{{/each}}
+
+{{/if}}
+{{#if module.events.length}}
+#### Events
+{{#each module.events}}
+- {{documentation}}
+{{/each}}
+
+{{/if}}
+{{#if @first}}
```

Comment:
Instead of flattening the array can we have 2 arrays? One for component and another for framework specific modules.

Reply:
Hmm you mean sort by adapter/foundation and component?

Reply:
the template data assumes that first item in modules array is always `component.ts` module. It would be nice if we've separate entry to reference component APIs and Framework specific APIs, so that the order in the array doesn't matter. WDYT?

Reply:
yeah i agree it is fragile...let me look into that.